roles/openshift_logging/defaults/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+image_prefix: docker.io/openshift/origin-
+image_version: latest
+logging_use_ops: False
+master_url: https://kubernetes.default.svc.cluster.local
+public_master_url: https://localhost:8443
+logging_namespace: logging
+etcd_generated_certs_dir: /etc/etcd/generated_certs
+install_logging: True
+
+curator_default_days: 30
+curator_run_hour: 0
+curator_run_minute: 0
+curator_run_timezone: UTC
+curator_script_log_level: INFO
+curator_log_level: ERROR
+
+kibana_hostname: kibana.example.com
+kibana_ops_hostname: kibana-ops.example.com
```

Comment:
I had played around with the idea that we would have variables that are used across all components to be in main.yml and ones specific to kibana would be in kibana.yml; however I'm not sure if there's a good case to be made to split them up.

I think we would have all of our defaults in main.yml and possibly create a second defaults file called 'development' or something, which we would keep dev-env/PoC settings in.